Packs/mnemonicMDR/IncidentTypes/incidenttype-Argus_Case.json

```diff
@@ -1,24 +1,26 @@
-{
- "id": "Argus Case",
- "version": -1,
- "vcShouldIgnore": false,
- "locked": false,
- "name": "Argus Case",
- "prevName": "Argus Case",
- "color": "#ff7500",
- "playbookId": "Argus Pull Case",
- "hours": 0,
- "days": 0,
- "weeks": 0,
- "hoursR": 0,
- "daysR": 0,
- "weeksR": 0,
- "system": false,
- "readonly": false,
- "default": false,
- "autorun": false,
- "disabled": false,
- "reputationCalc": 0,
- "layout": "Argus Case",
- "fromVersion": "5.0.0"
-}
\ No newline at end of file
+[
```

Comment:
Where did this change come from? please revert

Reply:
Right. This is due to a discrepancy in the format between what XSOAR accepts when you try to import this object, and what the sdk sees as "valid."